intrastat_product/models/account_move.py

```diff
@@ -158,11 +171,23 @@ class AccountMoveIntrastatLine(models.Model):
     transaction_weight = fields.Integer(
         help="Transaction weight in Kg: Quantity x Product Weight"
     )
+    # product_origin_country_id is replaced by product_origin_country_code
+    # this field should be dropped once the localisation modules have been
+    # adapted accordingly
     product_origin_country_id = fields.Many2one(
         comodel_name="res.country",
         string="Country of Origin",
         help="Country of origin of the product i.e. product " "'made in ____'.",
     )
+    product_origin_country_code = fields.Char(
+        string="Country of Origin of the Product",
+        size=2,
+        required=True,
```

Comment:
What happen with existing lines not having this filled?

Reply:
I think it is a strange behavior, but it is what was added in v14. In my opinion, this should be a compute store field. What do you think about this?

Reply:
We should keep the changes at minimum to comply with the Brexit requirement

Reply:
The `product_origin_country_code` field was added by the v14 PR to add the Brexit requirement (it is not country-related because it depends on the state).

You can change that field and make it a compute store, although it would be necessary to add a `product_origin_state_id` field as well (this would avoid overwriting the create() and write() of the account.move).
In any case, migration scripts will be necessary. Which way do you think is better?